tests/mapping/swagger20_request_validator/enum_validator_test.py

```diff
@@ -0,0 +1,30 @@
+import mock
+
+from bravado.mapping.swagger20_validator import enum_validator
+
+
+def test_multiple_jsonschem_call_if_enum_items_present_as_array():
+    enums = ['a1', 'b2', 'c3']
+    param_schema = {
+        'name': 'foo',
+        'in': 'query',
+        'type': 'array',
+        'items': {'type': 'array'},
```

Comment:
Do you mean for this to be `'items': {'type': 'string'},`  ?  (even though not really relevant to the test)